prisma/schema.prisma

```diff
@@ -0,0 +1,32 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./database.db"
+}
+
+model Warn {
+  id        Int      @id @default(autoincrement())
```

Comment:
Minor thing but there's no reason for these to not be unsigned

Reply:
sqlite does not have unsigned integers as a distinct data type